presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/SchemaDaoUtil.java

```diff
@@ -57,6 +57,22 @@ private static void createTables(SchemaDao dao)
         dao.createTableDeletedShards();
         dao.createTableBuckets();
         dao.createTableShardOrganizerJobs();
+
+        // for upgrading compatibility
+        alterTable(dao::alterTableTablesWithDeltaDelete);
+        alterTable(dao::alterTableTablesWithDeltaCount);
+        alterTable(dao::alterTableShardsWithIsDelta);
+        alterTable(dao::alterTableShardsWithDeltaUuid);
+    }
+
+    private static void alterTable(Runnable alterTableFunction)
+    {
+        try {
+            alterTableFunction.run();
+        }
+        catch (Exception e) {
+            log.warn(e.getMessage(), "Alter table failed");
```

Comment:
This isn't about passing Travis. The statement itself is incorrect. There is no formatting in `e.getMessage()`. If it prints too many lines, just put "// no-op" here without doing anything.